Dockerfile

```diff
@@ -47,10 +47,10 @@ RUN --mount=type=cache,target=/root/.cache/go-build \
 #
 FROM golang:1.22 AS runner
 
-RUN apt-get update     && \
-  DEBIAN_FRONTEND=noninteractive apt-get install -yq --no-install-recommends \
-  dnsutils=1:9.18.24-1 && \
-  rm -rf /var/lib/apt/lists/*
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -yq --no-install-recommends \
+    dnsutils && \
```

Comment:
the version should be hardcoded as this was a request from the last audit

you can find the correct latest version by running that golang image and using the apt-get command inside